modules/shared/components/molecules/Navigation/Navigation.module.css

```diff
@@ -17,3 +17,7 @@
 .nav-links li.nav-user {
   @apply md:mr-0;
 }
+
+.search-box {
+  @apply hidden md:block ml-l w-19xl bg-grey-bg border border-grey-shd5 rounded-md;
+}
```

Comment:
A small issue here, the search bar is not centered with the logo as it is in the design, you may need to adjust it a little.
this is the design
![Capture](https://user-images.githubusercontent.com/68955535/129450439-25785f1d-7ca5-4bb1-a7b9-13fc12e99358.PNG)
this is the pr
![Capture](https://user-images.githubusercontent.com/68955535/129450473-59e98e19-1b72-41f2-b772-2c8ec62cb6b3.PNG)

Reply:
requested changes applied